WWDCFamily/Source/Main/AppDelegate.swift

```diff
@@ -20,11 +28,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
-    func authViewController() -> AuthViewController {
+    func logOut() {
```

Comment:
Let's keep application logic out of the AppDelegate. We should setup a root navigation controller that will house the progression from an authenticated state to the main map viewcontroller